src/transforms/push.ts

```diff
@@ -64,12 +65,13 @@ export const createJobData = (payload, jiraHost: string): PushQueueMessagePayloa
 		jiraHost,
 		installationId: payload.installation.id,
 		webhookId: payload.webhookId || "none",
-		webhookReceived: payload.webhookReceived || undefined
+		webhookReceived: payload.webhookReceived || undefined,
+		...(!gitHubAppConfig ? undefined : { gitHubAppConfig })
```

Comment:
/nit **I would double check** but you should be able to simplify it a bit with
`...(gitHubAppConfig) && { gitHubAppConfig }`

Reply:
I can do you one better with just adding `githubAppConfig`.  if it's undefined, it'll be `githubAppConfig: undefined`, if not undefined, it'll show the object.  The only reason this might not work correctly is if the config returns null (which it shouldn't) and you could do `githubAppConfig: githubAppConfig || undefined` if you needed to handle that

Reply:
Just fixed it in the simplest way. I must be over thinking on `{ }` vs `{ gitHubConfig: undefined}`